python/GafferSceneTest/CollectScenesTest.py

```diff
@@ -129,23 +129,23 @@ def test( self ) :
 	def testGlobals( self ) :
 
 		options = GafferScene.CustomOptions()
-		options["options"].addChild( Gaffer.NameValuePlug( "user:test", "${collect:rootName}" ) )
-		self.assertTrue( "option:user:test" in options["out"]["globals"].getValue() )
+		options["options"].addChild( Gaffer.NameValuePlug( "render:test", "${collect:rootName}" ) )
```

Comment:
Just noting that these are options rather than attributes - will discuss further elsewhere.